pkg/serverless/logs/logs_collector.go

```diff
@@ -279,6 +283,22 @@ func (lc *LambdaLogsCollector) processMessage(
 				outOfMemoryRequestId = lc.lastRequestID
 			}
 		}
+
+		// Bottlecap Failover
+		if message.logType == logTypeExtension {
+			var r map[string]interface{}
+			record := message.stringRecord
+			err := json.Unmarshal([]byte(record), &r)
```

Comment:
Not fan of the idea of trying to unmarshal every extension log just to find the reason, is there a faster way of doing this?

Reply:
ah yeah that's a bummer, wondering if we can grep for the env var and parse if we get a hit?

Reply:
nice fix, looks good!